src/service/viewport/viewport-binding-natural.js

```diff
@@ -222,7 +222,12 @@ export class ViewportBindingNatural_ {
 
   /** @override */
   getLayoutRect(el, opt_scrollLeft, opt_scrollTop) {
-    const b = el./*OK*/getBoundingClientRect();
+    // Default value for when 'el' is has not been added to the page yet.
+    let b = 0;
+    if (el.isConnected) {
```

Comment:
From chat:

> JP: hey I think that Node.isConnected is not supported by IE

> Will: Correct 🙂 but we should use it if it's available
Another hint: try searching through current codebase to see if there's precedence
Always try to find any historical context in code first